build/Stride.Build.props

```diff
@@ -2,9 +2,14 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <StrideCommonPreSettingsName>Stride</StrideCommonPreSettingsName>
-    <StridePlatforms Condition="'$(StridePlatforms)' == ''">Windows</StridePlatforms>
+    <StridePlatforms Condition="'$(StridePlatforms)' == '' And '$([MSBuild]::IsOSPlatform(Windows))'">Windows</StridePlatforms>
+    <StridePlatforms Condition="'$(StridePlatforms)' == '' And '$([MSBuild]::IsOSPlatform(Linux))'">Linux</StridePlatforms>
```

Comment:
Same remark.

Reply:
I would prefer then without spacing 👀

Reply:
I dont think so, because is only chooses ONE platform. For Linux StridePlatforms should be 'Linux', for Windows 'Windows' etc. 

When release built is made StridePlatforms is collection of ALL platforms : 'Windows, Linux, MacOS, Android, iOS, UWP' 

So this is a default value for testing packages locally without worring about other platforms